app/src/main/java/io/homeassistant/android/view/viewholders/SwitchViewHolder.java

```diff
@@ -43,8 +44,8 @@ protected void updateViews() {
         super.updateViews();
         stateSwitch.setChecked(entity.state.equals(HassUtils.getOnState(entity, true)));
         stateSwitch.setOnClickListener(this);
-        if ((entity.attributes.supported_features & Common.LIGHT_SUPPORTS_BRIGHTNESS) == Common.LIGHT_SUPPORTS_BRIGHTNESS) {
-            brightnessSlider.setProgress((int) entity.attributes.brightness);
+        if ((entity.attributes.getInt("supported_features",0) & Common.LIGHT_SUPPORTS_BRIGHTNESS) == Common.LIGHT_SUPPORTS_BRIGHTNESS) {
```

Comment:
No default value required, `getInt(String)` is a shorthand for `getInt(String, 0)`.